src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/MyGroupCollectionAttribute.vb

```diff
@@ -0,0 +1,81 @@
+' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System
+Imports System.ComponentModel
+
+Namespace Microsoft.VisualBasic
+    ''' <summary>
+    ''' This attribute is put on an empty 'container class' that the compiler then fills with
+    ''' properties that return instances of all the types found in the project which derive
+    ''' from the TypeToCollect argument.
+    ''' 
+    ''' This is how My.Forms is built, for instance.
+    ''' </summary>
+    ''' <remarks>
+    ''' WARNING: Do not rename this attribute or move it out of this module.  Otherwise there
+    ''' are compiler changes that will need to be made
+    ''' </remarks>
+    <AttributeUsage(AttributeTargets.Class, AllowMultiple:=False, Inherited:=False)> _
+    <EditorBrowsable(EditorBrowsableState.Advanced)> _
+    Public NotInheritable Class MyGroupCollectionAttribute : Inherits Attribute
+
+        Private m_NameOfBaseTypeToCollect As String ' Do not rename - the runtime relies on this name.
```

Comment:
Interesting. I wonder what it means by "runtime" here? I do not see 'm_NameOfCreateMethod' in Roslyn or CoreCLR. I don't see it in internal NETFX sources either. @OmarTawfik  ?

Reply:
I meant to comment this actually - I sort of lied here as I don't know... I don't have the code for the runtime but I couldn't find any uses of this attribute so assumed its like some of the interop code where the field name is special

Reply:
I can't find it in the VS sources either (although I may be looking in the wrong repo). Hopefully @OmarTawfik can help. I am mainly interested in whether there is missing compiler or runtime work.

Reply:
Didn't find anything either :(
Perhahs @AlekseyTs?

Reply:
Possibly any dependency vanished 15 years ago and since then nobody has dared to remove it. Maybe this is our opportunity to remove it and see if anything breaks.